src/test/java/browserstack/utils/BrowserstackTestStatusListener.java

```diff
@@ -0,0 +1,36 @@
+package browserstack.utils;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.testng.ITestResult;
+import org.testng.TestListenerAdapter;
+import browserstack.stepdefs.BaseTest;
+
+
+public class BrowserstackTestStatusListener extends TestListenerAdapter {
+
+    private void markTestStatus(String status, String reason, WebDriver driver) { 
+        try {
+            JavascriptExecutor jse = (JavascriptExecutor) driver;
+            jse.executeScript("browserstack_executor: {\"action\": \"setSessionStatus\", \"arguments\": {\"status\": \"" + status + "\", \"reason\": \"" + reason + "\"}}");
+        } catch (Exception e) {
+            System.out.print("Error executing javascript");
```

Comment:
when running tests from docker this always lands up in the "catch" block and prints this error, maybe we can handle it in a better way